Domain/Repository/IBorrowRepository.cs

```diff
@@ -4,7 +4,7 @@ namespace Opcion1LosBorbotones.Domain.Repository;
 
 public interface IBorrowRepository : IRepository<Borrow>
 {
-    Task<IEnumerable<Borrow>> GetBorrowsByPatron(Guid patronId, int offset, int limit);
+    Task<IEnumerable<Borrow>> GetBorrowsByPatron(long membershipNumber, int offset, int limit);
```

Comment:
What are the advantages of using long instead of Guid

Reply:
More than advantages, it would be that, as the name says, we will ask the user for their membership number so that they no longer have to enter the ID as was previously requested by the console.

Reply:
nice